app/views/staff/organisations/show.html.haml

```diff
@@ -8,6 +8,31 @@
       %h1.govuk-heading-xl
         = @organisation_presenter.name
 
+  - if policy(:fund).index? & @organisation_presenter.service_owner?
+    .govuk-grid-row
+      .govuk-grid-column-full
+        %h2.govuk-heading-m Funds
+
+        - if policy(:fund).create?
+          = form_tag(organisation_funds_path(@organisation_presenter), method: "post") do
+            = submit_tag t("page_content.organisation.button.create_fund"), class: "govuk-button"
+
+        = render partial: "staff/shared/funds/table", locals: { funds: @fund_activities }
+
+  - if policy(:programme).index?
+    .govuk-grid-row
+      .govuk-grid-column-full
+        %h2.govuk-heading-m Programmes
+
+        = render partial: "staff/shared/programmes/table", locals: { programmes: @programme_activities }
+
+  - if policy(:project).index?
+    .govuk-grid-row
+      .govuk-grid-column-full
+        %h2.govuk-heading-m Projects
+
+        = render partial: "staff/shared/projects/table", locals: { projects: @project_activities }
+
   .govuk-grid-row
     .govuk-grid-column-two-thirds
       %h2.govuk-heading-m Details
```

Comment:
Internationalise me!

Also I think this should be "Organisation details" maybe? Now that the details are so far down the page and after the lists of funds/programmes, it's as if we've lost the context of what these details are.

Reply:
Yep, agree - good shout!